### PR TITLE
Feature 9 home window

### PR DESCRIPTION
**Pull request by dimitrov93**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,26 +1,124 @@
...
-}
+    NavBar,
+  },
+  data() {
+    return {
+      pages: [
+        {
+          link: { text: "Home", url: "/" },
+          pageTitle: "Home Page",
+          content: "This is the home content",
```
Comment (VladoKat) [inline src/App.vue:23]: The pageTitle and content are not used in the button from what I saw. Can we remove them from here to not cause confusion?

File: src/components/PageViewer.vue
[the remark this answers is not in the document]
Comment (dimitrov93) [reply src/components/PageViewer.vue:1]: Yes, so we can see that clicking on the nav items something is changing

File: src/App.vue (hunk 1 above)
Comment (dimitrov93) [reply src/App.vue]: done

**Review by VladoKat — APPROVED** [review]
Looks good! 🏅

**Review by aleks247 — APPROVED** [review]
I tested everything, and it looks fine!

**Pull request merged**